app/core/rate_limiting/circuit_breaker.py

```
@@ -0,0 +1,193 @@
+                    f"Circuit breaker {self.name} is open",
+                    estimated_recovery=self.next_attempt_time,
+                    failure_count=self.failure_count
+                )
+        
+        try:
+            # Execute the function
+            result = await func(*args, **kwargs) if asyncio.iscoroutinefunction(func) else func(*args, **kwargs)
+            await self._on_success()
+            return result
+            
+        except Exception as e:
```

Ruff (0.11.9)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)